### ISSUE TITLE
Pyramid elements C3D5 and C3D13

### ISSUE THREAD

**Issue by 3rav** [Dhondtguido/CalculiX#76]
If you think it's worth it, please add support for pyramid elements: https://calculix.discourse.group/t/pyramid-elements-c3d5-c3d13/1881/16

fgr: "I’m from another decade :grinning: Learned and wrote my first program on a teletyper terminal with paper roll, so I don’t very much about github and push request :grinning: but if anyone find my code worthy for the official calculix I will be happy just to know that other finds it usefull."

**Comment by victorkemp** [comment]
Yes please! These pyramids work far better and more conveniently than collapsed hex's. Maybe don't need the C3D13F (Felippa) formulation but the other two would be fantastic.

**Comment by NorbertHofbauer** [comment]
Yes please! Pyramid elements would greatly help with mesh transitions from hex to tet!
